apps/site/assets/package.json

```diff
@@ -31,6 +31,7 @@
     "react-redux": "^7.2.0",
     "redux": "^4.0.5",
     "sifter": "^0.6.0",
+    "stickyfilljs": "^2.1.0",
```

Comment:
Is this why it works without the `fixedsticky` class? That's quite great if so!

Reply:
Yes, a few notes on that:
- `fixedsticky` was in general breaking the whole layout of the page, regardless of the "stickiness" not working for IE. I think we should create a sort-of-follow-up ticket to examine whether `fixedsticky` works _at all_ and/or is breaking other pages in IE (I remember @hannahpurcell mentioned she saw that for accordions? I'm not sure if I remember that correctly). If that's the case, we could get rid of it completely and use this `stickyfill` package instead.

- I just noticed that `stickyfill` has been recently stopped being maintained (for a reason that I don't think is true?). But the other alternative I found ([stickybits](https://github.com/yowainwright/stickybits)) I couldn't get to work 😓